plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHivePartitionsOnDataLake.java

```diff
@@ -32,17 +39,18 @@
 import static java.util.stream.Collectors.joining;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
-public abstract class BaseTestHiveInsertOverwrite
+public abstract class BaseTestHivePartitionsOnDataLake
         extends AbstractTestQueryFramework
 {
     private static final String HIVE_TEST_SCHEMA = "hive_insert_overwrite";
 
     private String bucketName;
     private HiveMinioDataLake dockerizedS3DataLake;
+    private HiveMetastore hiveMetastore;
```

Comment:
`metastoreClient` to hint it's not shared with connector under test (ie not pulled from connector's Guice injector)